content/web-frameworks/django/docker-compose-for-django-mysql.md

```diff
@@ -16,7 +16,7 @@ To use Django and MySQL Docker compose, we need to create these required files a
 - Example content of `Dockerfile`
 ```sh
 # Dockerfile
-FROM python:3
+FROM python:alpine3.14
```

Comment:
Here is optional comment, 

I think when we write python the developer need to know python version before writing code.
tag name of docker image should be self explanation for the reader.

in this case I found similar tag for `alpine3.14`, I think it should be `3.9-alpine3.14` that tell, "using python 3.9" for alpine 3.14

```
Ref From: https://hub.docker.com/_/python
3.9.6-alpine3.14, 3.9-alpine3.14, 3-alpine3.14, alpine3.14, 3.9.6-alpine, 3.9-alpine, 3-alpine, alpine
```